src/AMSlib/ml/uq.hpp

```diff
@@ -17,9 +17,7 @@
 #include "ml/surrogate.hpp"
 #include "wf/resource_manager.hpp"
 
-static inline bool isNullOrEmpty(const char *p) {
-  return (!p || p[0] =='\0');
-}
+static inline bool isNullOrEmpty(const char *p) { return (!p || p[0] == '\0'); }
```

Comment:
Format is off

Reply:
`clang-format --style=file -i src/AMSlib/ml/uq.hpp` makes this.